### ISSUE TITLE
No focus set on the Step 1 screen with Talkback on

### ISSUE THREAD

**Issue by sarasuri** [cds-snc/covid-shield-mobile#732]
Samsung Galaxy J3 Prime
Android version 7.0
App version: 1.0 (53)

With Talkback on, launch the app and after splash screen, no focus is set on the text "Step 1 of 5" like other subsequent screens on the on-boarding flow.

Expected result
The focus should be set on the text like other screens.
